roles/etcd/tasks/permission_directory.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Directory | check certificate permissions
+  file:
+    path: "{{ etcd_config_dir }}"
+    group: "{{ etcd_cert_group }}"
+    state: directory
+    owner: etcd
+    mode: "{{ etcd_cert_dir_mode }}"
+    recurse: yes
```

Comment:
Is Calico etcd still working after this ? We don't want to break people updating
`/etc/calico/certs/key.pem` is a hardlink to `/etc/ssl/etcd/ssl/node-HOSTNAME-key.pem`
Also no reasons to make files executable

Reply:
fair, not tested work calico, use clium

Reply:
fix only for: when etcd_deployment_type == "host"